files/fr/web/api/htmliframeelement/allowpaymentrequest/index.html

```diff
@@ -0,0 +1,27 @@
+---
+title: HTMLIFrameElement.allowPaymentRequest
+slug: Web/API/HTMLIFrameElement/allowPaymentRequest
+browser-compat: api.HTMLIFrameElement.allowPaymentRequest
+translation_of: 'HTMLIFrameElement.allowPaymentRequest'
+---
+<p>{{APIRef("HTML DOM")}}{{deprecated_header}}{{non-standard_header}}</p>
+
+<p>La propriété <strong><code>allowPaymentRequest</code></strong> de l'interface {{domxref("HTMLIFrameElement")}} retourne une valeur booléenne indiquant si la <a href="/fr/docs/Web/API/Payment_Request_API"><i lang="fr">Payment Request API</i></a> doit être invoquée sur une iframe en <code>cross-origin</code>.</p>
```

Comment:
"may" = "peut" (and not "doit")
I'd use "utilisant une origine tierce" and not "en &lt;code&gt;cross-origin&lt;/code&gt;"

Reply:
good point! 
Fixed in https://github.com/mdn/translated-content/pull/1792/commits/a5ce292d7daad64040727f847dd639d8249315db, which also fixed a mistake on one <i> language attribute.